jest/private/jest_test.bzl

```diff
@@ -42,6 +42,8 @@ def _impl(ctx):
         ctx.file.config.short_path,
         "--testSequencer",
         relative_file(ctx.file.sequencer.short_path, ctx.file.config.short_path),
+        "--reporters",
+        "jest-junit",
```

Comment:
This is not where we ultimately want this flag because the cli reporters options doesn't let you specify options (we need to point the output to `process.env.XML_OUTPUT_FILE`. I'll have to do some modifications to a users jest config file in the rule.